packages/editor/src/store/test/effects.js

```diff
@@ -257,17 +268,7 @@ describe( 'effects', () => {
 
 			handler( { post, previousPost, postType }, store );
 
-			expect( dispatch ).toHaveBeenCalledTimes( 1 );
-			expect( dispatch ).toHaveBeenCalledWith( expect.objectContaining( {
-				notice: {
-					content: <p>Post published.{ ' ' }<a>View post</a></p>, // eslint-disable-line jsx-a11y/anchor-is-valid
-					id: 'SAVE_POST_NOTICE_ID',
```

Comment:
To my point about value of constants, we don't even use them consistently as constants.

Reply:
Yes, let’s not use them at all then 😃